osu.Game.Rulesets.Tau/Objects/Drawables/DrawableTauHitObject.cs

```diff
@@ -22,7 +22,7 @@ protected DrawableTauHitObject(TauHitObject obj)
         };
 
         /// <summary>
-        /// The action that caused this <see cref="DrawableHit"/> to be hit.
+        /// The action that caused HitObjects to be hit.
```

Comment:
Any reason for this documentation change?

Reply:
It’s giving out an error that DrawableHit doesn’t exist, I might need to check on this again.

Reply:
Probably because there's no using for it anywhere.

Reply:
Ah, did not notice this, putting it in the next commit

Reply:
There seems to be a `DrawableHitObject` but not `DrawableHit`. Is it supposed to be this?

Reply:
I believe so, yes.

Reply:
I found the cause as to why dotnet thinks `DrawableHit` is missing.
The editor code has undergone some changes in osu.Game, and fixing that gives out no errors on this specific file as well as the `TauSelectionHandler.cs` file.